### PR TITLE
MindMatch feedback

### PR DESCRIPTION
**Pull request by RoxasShadow**
Include the data for sending feedback to MindMatch.

### CODE REVIEW

File: src/resources/score.rs [hunk 1]
```diff
@@ -23,7 +23,9 @@ pub struct SearchResults {
 /// The representation of the score that will be indexed into ElasticSearch.
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct Score {
-  pub match_id:  String,
+  pub request_id:  String,
+  pub person_id:   String,
+  pub position_id: String,
```
Comment (kdamball) [inline src/resources/score.rs]: We also need [company_id](https://github.com/honeypotio/honeypot/blob/17631f539f186b5958b25f80f2484bf7c7d30a9f/frontend/app/pods/components/talent-profile/details/template.hbs#L47)

Comment (RoxasShadow) [reply src/resources/score.rs]: Can't we just take it from the FE?

Comment (kdamball) [reply src/resources/score.rs]: But it's not honeypot's company id. It's mindmatch's company id.

Comment (duksis) [reply src/resources/score.rs]: sadly but Kado is right :(

Comment (RoxasShadow) [reply src/resources/score.rs]: Done :)

**Pull request merged**